frontend/src/AutocompleteInput.svelte

```diff
@@ -74,6 +74,9 @@
     } else if (event.key === " " && event.ctrlKey) {
       hidden = false;
     } else if (event.key === "Escape") {
+      if (!hidden && filteredSuggestions.length > 0) {
```

Comment:
Checking `!hidden` is insufficient, as the suggestion list is also hidden when there are no suggestions.